src/App.css

```diff
@@ -0,0 +1,322 @@
+@import url('https://fonts.googleapis.com/css?family=Oxygen:400|Work+Sans|Open+Sans|Nanum+Gothic:400');
+
+body {
+  height: 100%;
+  width: 100%;
+  background-color: #080808;
+  font-family: 'Nanum Gothic';
+}
+
+h1, h2, h3, h4, h5, h6, a {
+  font-family: 'Nanum Gothic';
+  color: white;
+  text-decoration: none;
+}
+
+a:hover{
+  color: #52b0f8;
+  cursor: pointer;
+}
+
+.blue {
+  color: #52b0f8;
+}
+
+.pink {
+  color: #eb4a9d;
+} 
+
+.yel {
+  color: #fbe5a0;
+}
+
+.app {
+  color: white;
+  margin-left: 200px;
+  margin-right: 200px;
+  margin-top: 40px;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 40px;
+  border-bottom: 4px solid #eb4a9d;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+  animation: fade-in 1s cubic-bezier(0.390, 0.575, 0.565, 1.000) both;
+}
+
+.logo {
+  height: 60px;
+  width: auto;
+  animation: float 4s ease-in-out infinite;
+}
+
+.navbar-item {
+  margin-left: 20px;
+  font-size: 22px;
+  position: relative;
+  top: 0px;
+  transition: top ease 0.25s;
+}
+
+.navbar-item:hover {
+  top: -5px;
+}
+
+.footer {
+  text-align: end;
+  animation: slide-in ease-in-out 1.3s
+}
+
+.navbar-item, .big-logo h1, .subscribe button {
+  font-family: 'Oxygen';
+}
+
+/* main page */
+
+.intro-container {
+  display: flex;
+  justify-content: space-evenly;
+  padding-bottom: 50px;
+  margin-bottom: 40px;
+  border-bottom: 4px solid #eb4a9d;
+  border-bottom-right-radius: 5px;
+  border-bottom-left-radius: 5px;
+  animation: slide-in 1.3s ease-in-out both;
+}
+
+.big-logo {
+  text-align: center;
+  font-size: 36px;
+}
+
+.big-logo > img {
+  height: 300px;
+}
+
+.intro {
+  width: 50%;
+}
+
+.intro h2 {
+  color: #52b0f8;
+  font-size: 36px;
+}
+
+.intro p {
+  font-size: 22px;
+}
+
+.subscribe {
+  display: flex;
+  margin-top: 60px;
+}
+
+.subscribe button {
+  display: flex;
+  align-items: middle;
+  margin-right: 20px;
+  padding: 12px 12px;
+  border: 0 none;
+  border-radius: 4px;
+  text-align: center;
+  transition: all  150ms linear;
+
+  font-size: 22px;
+  color: white;
+  
+  cursor: pointer;
+  user-select: none;
+}
+
+.subscribe button img {
+  padding-left: 10px;
+  height: 32px;
+  width: auto;
+}
+
+.subscribe button:hover {
+  opacity: 0.85;
+  transform: scale(1.1);
+
+}
+
+.subscribe .mail-button {
+  background-color: #52b0f8;
+}
+
+.subscribe .gitter-button {
+  background-color: #eb4a9d;
+}
+
+.team-container {
+  margin-left: 1%;
+  margin-right: 1%;
+  animation: slide-in 1.3s ease-in-out 0.75s both;
```

Comment:
Here too, I think these big animations take away from the page or are distracting.